torch/nn/parallel/data_parallel.py

```diff
@@ -6,8 +6,8 @@
 from .scatter_gather import scatter_kwargs, gather
 from .replicate import replicate
 from .parallel_apply import parallel_apply
-from torch.cuda._utils import _get_device_index
-
+from torch._utils import _get_available_device_type, _get_device_index,\
```

Comment:
nit: recent changes in torch uses the following format if the import list is too long. Note that they are sorted using ascii order.

```python
from torch._utils import (
    _get_all_device_indices,
    _get_available_device_type,
    _get_device_index,
    _get_devices_properties,
)
```

Reply:
This also applies to other files.